src/components/application_manager/src/commands/mobile/perform_audio_pass_thru_request.cc

```diff
@@ -193,21 +196,35 @@ bool PerformAudioPassThruRequest::PrepareResponseParameters(
     mobile_apis::Result::eType& result_code, std::string& info) {
   LOG4CXX_AUTO_TRACE(logger_);
 
+  bool result = true;
   ResponseInfo ui_perform_info(result_ui_, HmiInterfaces::HMI_INTERFACE_UI);
   ResponseInfo tts_perform_info(result_tts_speak_,
                                 HmiInterfaces::HMI_INTERFACE_TTS);
-  const bool result =
-      PrepareResultForMobileResponse(ui_perform_info, tts_perform_info);
+
+  PrepareResultForMobileResponse(ui_perform_info, tts_perform_info);
```

Comment:
@dtrunov 
As the new CRQ defines new result code resolution (in the common case UI result code is with higher priority and must be transfered to the mobile side), the success result that is returned from PrepareResultForMobileResponse is not the right one in many cases and it must be changed.